spec/spec_helper.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'coveralls'
+
+Coveralls.wear!
+
+require 'bundler/setup'
+require 'rails'
+require 'active_record'
+require 'fibered_mysql2'
+
+RSpec.configure do |config|
+  # Enable flags like --only-failures and --next-failure
+  config.example_status_persistence_file_path = ".rspec_status"
+
+  # Disable RSpec exposing methods globally on `Module` and `main`
+  # config.disable_monkey_patching!
```

Comment:
Is it possible for us to go ahead and disable monkey patching in the tests? I think there are also some other configs that we generally set that would be good here.

```ruby
config.expect_with :rspec do |expectations|
  expectations.include_chain_clauses_in_custom_matcher_descriptions = true
end
config.mock_with :rspec do |mocks|
  mocks.verify_partial_doubles = true
end
```